packages/ra-core/src/controller/ListController.tsx

```diff
@@ -70,9 +145,29 @@ import checkMinimumRequiredProps from './checkMinimumRequiredProps';
  *         </List>
  *     );
  */
-export class ListController extends Component {
+export class ListControllerView extends Component<Props> {
+    public static defaultProps: Partial<Props> = {
+        debounce: 500,
+        filter: {},
+        perPage: 10,
+        sort: {
+            field: 'id',
+            order: SORT_DESC,
+        },
+    };
+
     state = {};
 
+    setFilters = debounce(filters => {
```

Comment:
why did you move that function?

Reply:
That's not me, tslint did it. I suppose it's because it considers it a private field and not a private function